server/app.py

```diff
@@ -1,20 +1,16 @@
+from .graphql import app as graphql_app
 import time
-
 from bocadillo import App, Templates
 
 from . import settings
 
+
 app = App()
+app.mount("/graphql", graphql_app)
```

Comment:
Since the `/graphql` prefix is defined both here and in `TartifletteApp`, am I correct saying the GraphQL endpoint is served at `/graphql/graphql`?

If mounting at `/` collides with the `index` route, perhaps removing `path="/graphql"` in `graphql.py` could do the trick?